Resources/Prototypes/Loadouts/Jobs/Engineering/chief_engineer.yml

```diff
@@ -8,6 +8,16 @@
       role: JobChiefEngineer
       time: 72000 #20 hrs
 
+# For the veteran command cosmetic set
+- type: loadoutEffectGroup
+  id: VeteranCommand
+  effects:
+  - !type:JobRequirementLoadoutEffect
```

Comment:
You've copied this object across multiple files with the same ID, which is causing YML errors. Please define it once and delete every other instance of it.

Reply:
Ah, my bad! I just jumped straight into the "Files Changed" tab 😅